README.md

```diff
@@ -1,33 +1,82 @@
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
 
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
+    - We were one of the [first companies to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
+    - We were one of the [first legacy companies to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
+    - We were a pioneer of the [concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
+ - We maintain our technical advantage through continuous innovation
+ - We aim to be at the forefront of tech, staying curious, looking to innovate and continuously improve.
 
+**Knowledge sharing**
 
+We believe that it’s our responsibility to upskill one another, sharing our knowledge, learnings and mistakes. 
 
+_Behaviours_:
+ - We practice pairing to help share our knowledge, grow our skills, and facilitate collaborative decision making.
+ - We value pairing as a skill which we encourage people to grow, working together to find the styles which are most effective.
+ - We make time for individual-learning & career development with personal development being our priority.
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```

Comment:
To my mind, it would be better to shorten the first sentence:

```suggestion
We believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. We are enthused to do whatever we can to deliver value to the Guardian.
```

Whether our values are _the_ reason we choose to work here is neither here nor there. If they are then that's fine, but I think there are a lot of other good reasons (many of them mentioned above) that one might choose to work here.